bp_common/test/cfg/linux_defconfig

```diff
@@ -0,0 +1,3005 @@
+#
```

Comment:
can we not commit this? autogenerated?

Reply:
Not fully autogenerated, it's generated by Linux's configuration menu.

Reply:
Hm.  Do the other SDKs (Ariane, freedom) save this output, or do they generate it in the build somehow?

Reply:
Most of them have a starting config file. I can clean it up later to see which flags are necessary to input to Linux generator and which ones are generated by default but it's out of the bounds of my knowledge right now.